Controller/Notify_Handler.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Controller;
+
+use \Wirecard\Oxid\Extend\Order;
+use \Wirecard\Oxid\Core\Payment_Method_Factory;
+use \Wirecard\Oxid\Model\Transaction;
+
+use \Wirecard\PaymentSdk\Response\Response;
+use \Wirecard\PaymentSdk\TransactionService;
+use \Wirecard\PaymentSdk\Exception\MalformedResponseException;
+use \Wirecard\PaymentSdk\Response\SuccessResponse;
+
+use \OxidEsales\Eshop\Application\Model\Order as Oxid_Order;
+use \OxidEsales\Eshop\Application\Controller\FrontendController;
+use \OxidEsales\Eshop\Core\Registry;
+use \OxidEsales\Eshop\Core\Field;
+
+use \Psr\Log\LoggerInterface;
+
+/**
+ * Notify handler class.
+ *
+ * Handle Payment SDK notifications.
+ */
+class Notify_Handler extends FrontendController
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $_oLogger;
+
+    /**
+     * NotifyHandler constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->_oLogger = Registry::getLogger();
+    }
+
+    /**
+     * Request handling function.
+     *
+     * @return void
+     */
+    public function handleRequest()
+    {
+        $sPaymentName = Registry::getRequest()->getRequestParameter('pmt');
+        $oPaymentMethod = Payment_Method_Factory::create($sPaymentName);
+        $oConfig = $oPaymentMethod->getConfig();
+        $sPostData = file_get_contents('php://input');
+
+        try {
+            $oService = new TransactionService($oConfig, $this->_oLogger);
+            $oNotificationResponse = $oService->handleNotification($sPostData);
+        } catch (\InvalidArgumentException $exception) {
```

Comment:
Please `use` this exception as well, to be consistent with the `catch` block below.